### PR TITLE
Update flows-config-schema.json

### PR DESCRIPTION
**Pull request by spokenbird**
[#185752154]

### CODE REVIEW

**Comment by bseeger** [comment]
Hi @spokenbird - can you explain how to test out this change? I applied the file, as described in our readme, and am not seeing it correct me. Not sure I'm doing it correctly. 
Thanks!

**Comment by bseeger** [comment]
It started working now.  I am seeing the auto complete.

File: intellij-settings/flows-config-schema.json [hunk 1]
```diff
@@ -43,14 +33,30 @@
                 }
               }
             },
-            "subflow": {
+            "onPostAction": {
               "type": "string",
-              "description": "For screen within a subflow, they need the name of the subflow defined."
+              "description": "A method name to run on a POST."
```
Comment (bseeger) [inline intellij-settings/flows-config-schema.json:38]: I think we should fill this description out a little more.   Something along the lines of "This is is the name of an Action method that will be called when a POST occurs, before any general data processing happens."

I think you can say Action here, so folks can link this back to what an Action is in the docs.

File: intellij-settings/flows-config-schema.json [hunk 2]
```diff
@@ -43,14 +33,30 @@
...
+            },
+            "crossFieldValidationAction": {
+              "type": "string",
+              "description": "A method name to to validate one field based on another field."
```
Comment (bseeger) [inline intellij-settings/flows-config-schema.json:42]: "An Action that will be called after general validation is done on the fields, but before the data is saved to the database.  It is a chance to do any cross field validation necessary."

Comment (bseeger) [reply intellij-settings/flows-config-schema.json:42]: I'd also fill the rest out some more as well.  If this is the only documentation that a SNE sees, then it's good if we give a little more context.

File: intellij-settings/flows-config-schema.json [hunk 3]
```diff
@@ -43,14 +33,30 @@
...
             },
-            "beforeSave": {
+            "beforeSaveAction": {
```
Comment (bseeger) [inline intellij-settings/flows-config-schema.json:44]: - before saving, but after validation
 - going point for

File: intellij-settings/flows-config-schema.json (hunk 2 above)
Comment (spokenbird) [reply intellij-settings/flows-config-schema.json]: Done!

**Review by bseeger — APPROVED** [review]
Looks good! Thanks for the updates!

**Pull request merged**